playwright-tests/tests/proposals.spec.js

```diff
@@ -455,4 +455,45 @@ test.describe("Wallet is connected", () => {
 
     await pauseIfVideoRecording(page);
   });
+
+  test("should filter proposals by categories", async ({ page }) => {
+    test.setTimeout(120000);
+    await page.goto("/devhub.near/widget/app?page=proposals");
+    await page.getByRole("button", { name: "Category" }).click();
+    await page.getByRole("list").getByText("DevDAO Operations").click();
+    await expect(
+      page.getByRole("button", { name: "Category : DevDAO Operations" })
+    ).toBeVisible();
+    test.setTimeout(4000);
+  });
+
+  test("should filter proposals by timeline", async ({ page }) => {
+    test.setTimeout(120000);
+    await page.goto("/devhub.near/widget/app?page=proposals");
+    await page.getByRole("button", { name: "Stage" }).click();
+    await page.getByRole("list").getByText("Funded").click();
+    await expect(
+      page.getByRole("button", { name: "Stage : Funded" })
+    ).toBeVisible();
+    test.setTimeout(4000);
+  });
+
+  test("should filter proposals by author", async ({ page }) => {
+    test.setTimeout(120000);
```

Comment:
Do we need 2 minutes timeout here?

Reply:
we have this timeout before every test, which I believe is to load the page